packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -0,0 +1,40 @@
+import {GraphQLList, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import IntegrationProviderOAuth1 from './IntegrationProviderOAuth1'
+import TeamMemberIntegrationAuthOAuth1 from './TeamMemberIntegrationAuthOAuth1'
+
+const JiraServerIntegration = new GraphQLObjectType<any, GQLContext>({
+  name: 'JiraServerIntegration',
+  description: 'Jira Server integration data for a given team member',
+  fields: () => ({
+    auth: {
+      description: 'The OAuth1 Authorization for this team member',
+      type: TeamMemberIntegrationAuthOAuth1,
+      resolve: async ({teamId, userId}, _args, {dataLoader}) => {
+        return dataLoader
+          .get('teamMemberIntegrationAuths')
+          .load({service: 'jiraServer', teamId, userId})
+      }
+    },
+    sharedProviders: {
+      description: 'The non-global providers shared with the team or organization',
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(IntegrationProviderOAuth1))),
+      resolve: async ({teamId, userId}, _args, {dataLoader}) => {
+        const teamMembers = await dataLoader.get('teamMembersByUserId').load(userId)
+        const teamMember = teamMembers.find((teamMember) => teamMember.teamId === teamId)
```

Comment:
+1 for some reason eslint complains here about `teamMember` having implicit `any` type here so you might want to check it (in general, PR added 4 more `no-implicit-any` issues)